src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrDependencyTracker.cs

```diff
@@ -103,67 +103,15 @@ public void UpdateDependencyVersion(VmrDependencyUpdate update)
             update.Mapping.Name,
             update.RemoteUri,
             update.TargetRevision,
-            update.TargetVersion,
             update.BarId);
         _fileSystem.WriteToFile(_vmrInfo.SourceManifestPath, _sourceManifest.ToJson());
-
-        var gitInfoDirPath = _vmrInfo.VmrPath / VmrInfo.GitInfoSourcesDir;
-        
-        // Only update git-info files if the git-info directory exists
-        if (!_fileSystem.DirectoryExists(gitInfoDirPath))
-        {
-            _logger.LogInformation("Skipped creating git-info files for {repo} as the git-info directory doesn't exist", update.Mapping.Name);
-            return;
-        }
-            
-        // Root repository of an update does not have a package version associated with it
-        // For installer, we leave whatever was there (e.g. 8.0.100)
-        // For one-off non-recursive updates of repositories, we keep the previous
-        string packageVersion = update.TargetVersion
-            ?? _sourceManifest.GetVersion(update.Mapping.Name)?.PackageVersion
-            ?? "0.0.0";
-
-        // If we didn't find a Bar build for the update, calculate it the old way
-        string? officialBuildId = update.OfficialBuildId;
-        if (string.IsNullOrEmpty(officialBuildId))
-        {
-            var (calculatedOfficialBuildId, _) = VersionFiles.DeriveBuildInfo(update.Mapping.Name, packageVersion);
-            officialBuildId = calculatedOfficialBuildId;
-        }
-
-        var gitInfo = new GitInfoFile
-        {
-            GitCommitHash = update.TargetRevision,
-            OfficialBuildId = officialBuildId,
-            OutputPackageVersion = packageVersion,
-        };
-
-        var gitInfoFilePath = GetGitInfoFilePath(update.Mapping);
-        gitInfo.SerializeToXml(gitInfoFilePath);
-        _logger.LogInformation("Updated git-info file {file} for {repo}", gitInfoFilePath, update.Mapping.Name);
     }
 
     public bool RemoveRepositoryVersion(string repo)
```

Comment:
also completely delete this method and it's use. There's an if/else in `VmrUpdater` using it, delete it